### DISCUSSION TITLE
Web Search prompts get stuck when Tika integration is enabled.

### DISCUSSION THREAD

**Discussion by SlOrbA** [open-webui/open-webui#3791] [Q&A]
I've enabled the PSE Web Search and Tika integrations to a local docker container. I can do RAG requests, but when I use the Web Search I don't get any response only timeout. Everything works again after I switch the "Content Extraction" to default from Tika.

**Comment by ProjectMoon** [comment]
I do not experience this with SearxNG and Tika. Worked fine.
